chia/wallet/wallet_state_manager.py

```diff
@@ -1592,11 +1592,29 @@ async def remove_from_queue(
         """
         Full node received our transaction, no need to keep it in queue anymore
         """
+
         updated = await self.tx_store.increment_sent(spendbundle_id, name, send_status, error)
         if updated:
             tx: Optional[TransactionRecord] = await self.get_transaction(spendbundle_id)
             if tx is not None:
-                self.state_changed("tx_update", tx.wallet_id, {"transaction": tx})
+                if send_status and error and error not in (Err.INVALID_FEE_LOW_FEE, Err.INVALID_FEE_TOO_CLOSE_TO_ZERO):
+                    if tx.spend_bundle is not None:
+                        coins_removed = tx.spend_bundle.removals()
```

Comment:
There's a cached removals property on TransactionRecord already

Reply:
unfortunately TransactionRecord.removals is not reliable (tests don't pass it I use it)